arrow-libs/core/arrow-core-data/src/main/kotlin/arrow/core/NonEmptyList.kt

```diff
@@ -526,11 +527,13 @@ inline fun <E, A, B> NonEmptyList<A>.traverseEither(f: (A) -> Either<E, B>): Eit
     f(a).ap(acc.map { bs -> { b: B -> NonEmptyList(b) + bs } })
   }
 
+@Deprecated(TraverseDeprecation)
```

Comment:
All of these didn't exist for `NonEmptyList` on `0.11.0` so can be removed.
https://github.com/arrow-kt/arrow-core/blob/0.11.0/arrow-core-data/src/main/kotlin/arrow/core/NonEmptyList.kt